<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Suite is getting ready]]>
    </line>
    <line>
      <![CDATA[Session has been ended]]>
    </line>
  </reporter-output>
  <suite started-at="2024-07-12T20:10:23 IST" name="Default suite" finished-at="2024-07-12T20:10:30 IST" duration-ms="7333">
    <groups>
    </groups>
    <test started-at="2024-07-12T20:10:23 IST" name="Default test" finished-at="2024-07-12T20:10:30 IST" duration-ms="7333">
      <class name="com.automation.tests.LoginPageTest">
        <test-method is-config="true" signature="setUpSuite()[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d]" started-at="2024-07-12T20:10:22 IST" name="setUpSuite" finished-at="2024-07-12T20:10:23 IST" duration-ms="750" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Suite is getting ready]]>
            </line>
          </reporter-output>
        </test-method> <!-- setUpSuite -->
        <test-method is-config="true" signature="setUp()[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d]" started-at="2024-07-12T20:10:23 IST" name="setUp" finished-at="2024-07-12T20:10:29 IST" duration-ms="6324" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="testLoginPage()[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d]" started-at="2024-07-12T20:10:29 IST" name="testLoginPage" finished-at="2024-07-12T20:10:29 IST" duration-ms="377" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertFalse(Assert.java:79)
at org.testng.Assert.assertFalse(Assert.java:89)
at com.automation.tests.LoginPageTest.testLoginPage(LoginPageTest.java:23)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLoginPage -->
        <test-method is-config="true" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d]" started-at="2024-07-12T20:10:29 IST" name="tearDownMethod" finished-at="2024-07-12T20:10:30 IST" duration-ms="588" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=testLoginPage status=FAILURE method=LoginPageTest.testLoginPage()[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.automation.tests.LoginPageTest@3c41ed1d]" started-at="2024-07-12T20:10:30 IST" name="tearDown" finished-at="2024-07-12T20:10:30 IST" duration-ms="201" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Session has been ended]]>
            </line>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.automation.tests.LoginPageTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
